docs/lib/auth/fragments/native_common/check_authentication_state/common.md

```diff
@@ -0,0 +1,6 @@
+Use the following method to check if the current user is authenticated.   This is the recommended approach for determining whether you should display a sign in screen when your application starts.
```

Comment:
suggestion

> Use the following method to check if the current user is authenticated.

Use the following method to check if the current user's authentication session is still valid.